src/sql/workbench/services/notebook/browser/notebookServiceImpl.ts

```diff
@@ -86,27 +85,28 @@ class ProviderDescriptor {
 	}
 }
 
+export const NotebookUriNotDefined = localize('notebookUriNotDefined', "No URI was passed when creating a notebook manager");
+
 export class NotebookService extends Disposable implements INotebookService {
 	_serviceBrand: undefined;
 
-	private _providersMemento: Memento;
-	private _trustedNotebooksMemento: Memento;
+	private _providersMemento: Memento = new Memento('notebookProviders', this._storageService);
```

Comment:
moved intialization of these fields from within constructor to here as was done with most other properties in this class for consistency.